iina/VideoView.swift

```diff
@@ -400,5 +405,16 @@ class VideoView: NSOpenGLView {
   func ignoreGLError() {
     glGetError()
   }
-
+  
+  override func draggingEntered(_ sender: NSDraggingInfo) -> NSDragOperation {
+    return .copy
+  }
+  
+  override func performDragOperation(_ sender: NSDraggingInfo) -> Bool {
+    let board = sender.draggingPasteboard()
+    let fileURL: NSURL? = NSURL(from: board)
+    playerCore.loadExternalSubFile(fileURL as! URL)
```

Comment:
Maybe need to check whether it's a subtitle file, if not, load the dragged file (as video).